disposable_email_blacklist.conf

```diff
@@ -420,6 +445,7 @@ cnew.ir
 cnmsg.net
 cnsds.de
 cobarekyo1.ml
+cock.li
```

Comment:
`cock.li` is in the `whitelist.conf`as it requires a registration before the service can be accessible